docs/RUBRIC.md

```diff
@@ -13,27 +13,28 @@ Key:
 
 |User Stories|Proposal A|Proposal B|Proposal C|Proposal D|
 |--- |--- |--- |--- |--- |
-|**F.1**| | | | |
-|**F.2**| | | | |
-|**F.3**| | | | |
-|**F.5**| | | | |
-|**F.6**| | | | |
-|**B.1**| | | | |
-|**B.2**| | | | |
-|**B.3**| | | | |
-|**B.4**| | | | |
-|**B.5**| | | | |
-|**B.6**| | | | |
-|**C.1**| | | | |
-|**C.2**| | | | |
-|**C.3**| | | | |
-|**C.4**| | | | |
-|**C.5**| | | | |
-|**C.6**| | | | |
-|**C.7**| | | | |
-|**C.8**| | | | |
-|**C.9**| | | | |
-|**C.10**| | | | |
-|**C.11**| | | | |
-|**C.12**| | | | |
-|**C.13**| | | | |
+|**F.1**| | |yes| |
+|**F.2**| | |yes| |
+|**F.3**| | |yes| |
+|**F.4**| | |no| |
+|**F.5**| | |yes| |
```

Comment:
I'm not sure how this would work if the originator of a bunch of signatures pushes them all together. But, if you want to push a new signature to a tag with a reference to the node manifest with the old signature, the most recent signature is the first node manifest from the tag.